app/styles/components/_footer.scss

```diff
@@ -13,46 +13,61 @@
     color: $color-white;
   }
 
+  .footer-service-guide {
+    text-align: center;
+    font-weight: 1000;
```

Comment:
Might be safer to use 900. I didn't actually realize that 1000 is a supported value now, but older browsers might only support 100-900. This chart from MDN doesn't even know if Edge or Safari support 1000: https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight#Browser_compatibility

Reply:
Done